src/jaxsim/api/contact.py

```diff
@@ -114,19 +114,23 @@ def collidable_point_forces(
 
 @jax.jit
 def collidable_point_dynamics(
-    model: js.model.JaxSimModel, data: js.data.JaxSimModelData
-) -> tuple[jtp.Matrix, jtp.Matrix]:
+    model: js.model.JaxSimModel,
+    data: js.data.JaxSimModelData,
+    link_external_forces: jtp.Matrix | None = None,
```

Comment:
Thank you for the explanation on the types, I will apply it my modifications. Concerning the "external" in the input argument, in this case I wanted to remark that they are in fact external and not contact forces, since the latter are computed in the body of the function and summed to the external link forces in input. Is it correct for you?

Reply:
Yep you are correct. In the past, we denotes as _external_ the user-applied link forces, but then we refactored to just `link_forces`. This is the current status of the APIs (check `jaxsim.api.Model`, `jaxsim.api.JaxSimModelReferences`, etc.). I'd remain consistent with them.